rbi/core/hash.rbi

```diff
@@ -1009,6 +1009,29 @@ class Hash < Object
   sig {returns(String)}
   def to_s(); end
 
+  # Return a new with the results of running block once for every value. This
+  # method does not change the keys.
+  #
+  # If no block is given, an enumerator is returned instead.
+  #
+  # ```ruby
+  # h = { a: 1, b: 2, c: 3 }
+  # h.transform_values {|v| v * v + 1 }  #=> { a: 2, b: 5, c: 10 }
+  # h.transform_values(&:to_s)           #=> { a: "1", b: "2", c: "3" }
+  # h.transform_values.with_index {|v, i| "#{v}.#{i}" }
+  #                                      #=> { a: "1.0", b: "2.1", c: "3.2" }
+  # ```
+  sig do
+    type_parameters(:A).params(
+      blk: T.proc.params(arg0: K).returns(T.type_parameter(:A))
```

Comment:
@marianosimone shouldn't this be `V`?

```suggestion
      blk: T.proc.params(arg0: V).returns(T.type_parameter(:A))
```

Reply:
Your tests don't have a test for this case.

Reply:
Fixed `K` -> `V`, and added a test that fails if the block passed in tries to call a method that does not exists in `V`